### ISSUE TITLE
Problem with removing legend in ggplot with geom_sf

### ISSUE THREAD

**Issue by neuwirthe** [tylermorganwall/rayshader#66]
I can do a very nice 3d object from a map with geom_sf using fill fr the height.
When I try to remove the legend with 
`scale_fill_continuous(guide=FALSE)
`
The map gets very messy

This is the correct map:

![2019-07-17_18-15-20](https://user-images.githubusercontent.com/157619/61392540-56345180-a8bf-11e9-8908-5afbfd68a1dd.png)

And that is what it looks like when I try to remove the legend:

![2019-07-17_18-15-47](https://user-images.githubusercontent.com/157619/61392593-72d08980-a8bf-11e9-9500-90070826162f.png)

This is the code to reproduce the problem:

```
load("guides_remain_incorrectly.RData")
library(tidyverse)
library(sf)
library(rayshader)

mapdata_vbg %>%
  ggplot(aes(fill=Dichte)) +
  geom_sf() +
  scale_x_continuous(breaks=c(0,90)) +
  scale_y_continuous(breaks=c(0,90)) +
  theme_minimal() ->
  gdichte_vbg

gdichte_vbg +
  scale_fill_continuous(guide=FALSE) ->
  gdichte_vbg_no_legend

gdichte_vbg

gdichte_vbg_no_legend

# plot_gg(gdichte_vbg)

plot_gg(gdichte_vbg_no_legend)
```

The file containing the data needed and the ggplot objects is enclosed:

[guides_remain_incorrectly.RData.zip](https://github.com/tylermorganwall/rayshader/files/3402859/guides_remain_incorrectly.RData.zip)

**Comment by tylermorganwall** [comment]
Fixed in 23906f5cc1c23532152a0c33ce07bb0f001ca812

**Comment by juanifulponi** [comment]
Hi, I have te same issue than @neuwirthe. How can I fix this installing that version? Sorry, I'm new :)

Thanks
